src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1053,5 +1084,32 @@ private void CompleteDatabaseUnloading(DocumentDatabase database)
 
             database.DatabaseShutdownCompleted.Set();
         }
+
+        private class IntersectionInPaths : Exception
```

Comment:
Why create a special exception for this?

Reply:
Done